addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -430,18 +430,17 @@ registry.slider = publicWidget.Widget.extend({
     selector: '.carousel',
     disabledInEditableMode: false,
     edit_events: {
-        'slid.bs.carousel': '_onEditionSlide',
+        'content_changed': '_computeHeights',
```

Comment:
Right on! Any idea why this uses strings for method names instead of just a function object?

Reply:
In fact you can use a function directly but this is currently a bad practice as that function will not be customizable by external developers (while he can `include` the widget to extend the `_on...` handler with this kind of definition)... but I guess all of this will change once we start using OWL.